apps/src/aichat/views/teacherFeedback/CleanFeedbackFooter.tsx

```diff
@@ -35,46 +41,51 @@ const CleanFeedbackFooter: React.FC<Props> = ({id, teacherFeedback, role}) => {
     );
   };
 
-  return (
```

Comment:
We were using `row-reverse` to sneaky make this work in both the Assistant and User message case. This doesn't play super nicely with tab navigation (ie, you tab first to buttons that are towards the interior of the row, which is unintuitive) and made the code a bit confusing (ie, I had to add the new copy button to the end of this HTML rather than the beginning, where I'd expect it to bed). I made the ordering explicit instead, LMK what you think.

Reply:
That makes sense, thanks for clearing this up!